portfolio/src/main/webapp/index.html

```diff
@@ -33,6 +33,9 @@ <h1 style="text-align:center"> Sean Tisdale Portfolio </h1>
             <p>Click here to get a random message:</p>
             <button onclick="addRandomFact()">Facts</button>
             <div id="fact-container"></div>
+            <br>
```

Comment:
nit: This 'br' and the one above it are at a different indentation than the one at line 24. Could you make them consistent?

Reply:
from looking at things they appear to already be the same indentation.

Reply:
Are you sure? I see them at different indentations - http://screen/7ASfxBrBbML.